vsphere/lib/vsphere/vclib/volumeoptions.go

```diff
@@ -34,6 +34,7 @@ type VolumeOptions struct {
 	StoragePolicyID        string
 	SCSIControllerType     string
 	Zone                   []string
+	DiskType               string
```

Comment:
Should we use `DiskMode` here? It will be more consistent as we are using the term mode elsewhere.

Reply:
yes, make sense. Changed it to DiskMode